bin/rpm-s3

```diff
@@ -272,6 +279,7 @@ if __name__ == '__main__':
     parser.add_option('-d', '--delete', action='store_true', default=False)
     parser.add_option('-r', '--region')
     parser.add_option('-c', '--host', default='s3-eu-central-1.amazonaws.com')
+    parser.add_option('-a', '--archs', type='string', action='callback', callback=get_comma_separated_args)
```

Comment:
The default value is `None` so the `for` loop above would fail if you don't pass the `--archs` value. I think this should fix it:
```suggestion
    parser.add_option('-a', '--archs', type='string', default=[], action='callback', callback=get_comma_separated_args)
```